test/integration/http_protocol_integration.h

```diff
@@ -60,76 +60,87 @@ class HttpProtocolIntegrationTest : public testing::TestWithParam<HttpProtocolTe
   }
 
 protected:
-  void expectUpstreamWireBytesSentAndReceived(std::string log_name, int log_index,
-                                              int h1_wire_bytes_sent, int h1_wire_bytes_received,
-                                              int h1_body_bytes_sent, int h1_body_bytes_received,
-                                              int h2_wire_bytes_sent, int h2_wire_bytes_received,
-                                              int h2_body_bytes_sent, int h2_body_bytes_received) {
+  struct BytesCountExpectation {
+    BytesCountExpectation(int wire_bytes_sent, int wire_bytes_received, int body_bytes_sent,
+                          int body_bytes_received)
+        : wire_bytes_sent_{wire_bytes_sent}, wire_bytes_received_{wire_bytes_received},
+          body_bytes_sent_{body_bytes_sent}, body_bytes_received_{body_bytes_received} {}
+    int wire_bytes_sent_;
+    int wire_bytes_received_;
+    int body_bytes_sent_;
+    int body_bytes_received_;
```

Comment:
Nit: Could follow the builder pattern with this so that this becomes even more readable when constructed.

e.g.
```
BytesCountExpectation& BytesCountExpectation::setWireBytesSent(int wire_bytes_sent) {
  wire_bytes_sent_ = wire_bytes_sent;
  return *this;
}
```

Which can then create as:
```
auto H1ExpectedBytes = BytesCountExpectation().setWireBytesSent(500).
                                        setBodyBytesSent(300).setWireBytesReceived(400).
                                        setBodyBytesReceived(200);
```